stack.yaml

```diff
@@ -25,13 +25,13 @@ extra-deps:
 - servant-auth-server-0.4.0.0
 
 - git: https://github.com/serokell/auth-data-structures.git
-  commit: f9a0cd66f443d7b0e04a3c8ae8d475f8344b823b
+  commit: 9167add1907456f62f6ed633228198875fd39207
```

Comment:
Sorry, i missed that -- so what's the strategy with this branch again?

Reply:
I will merge it into `auth-data-structures` master. But you need to go https://github.com/serokell/auth-data-structures/pull/10 and write your diss.